src/app/components/common/graphics-window/index.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from "react";
+import * as Konva from "konva";
+
+import "./style.css";
+
+interface GraphicsWindowComponentProps {
+
+}
+
+interface GraphicsWindowComponentState {
+  stage?: Konva.Stage;
+  layer?: Konva.Layer;
+}
+
+export class GraphicsWindowComponent extends React.Component<GraphicsWindowComponentProps, GraphicsWindowComponentState> {
+  private isAlreadyMounted: boolean;
+  private tokens: string[] = [];
+
+  public constructor(props: GraphicsWindowComponentProps) {
+    super(props);
+    this.isAlreadyMounted = false;
+
+    this.state = {};
+  }
+
+  public componentDidMount(): void {
+    this.tokens = [];
+
+    const stage = new Konva.Stage({
+      container: "graphics-container",
+      width: document.getElementById("graphics-container")!.offsetWidth,
```

Comment:
I know we were unsure, is there anyone who can advise on if this is the best way to get this? @shaminoo ?

Reply:
https://reactjs.org/docs/refs-and-the-dom.html